apm-sniffer/apm-sdk-plugin/mysql-5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/jdbc/mysql/v5/define/AbstractConnectionInstrumentation.java

```diff
@@ -41,7 +40,7 @@
  *
  * @author zhangxin
  */
-public abstract class ConnectionInstrumentation extends ClassInstanceMethodsEnhancePluginDefine {
+public abstract class AbstractConnectionInstrumentation extends AbstractMysqlInstrumentation {
```

Comment:
I suggest that the `AbstractConnectionInstrumentation` class should implement `withClasses` method, so the subclass of `AbstractConnectionInstrumentation`  could't implement this method

Reply:
this class  extends ``AbstractMysqlInstrumentation`` class witch has implement ``witnessClasses`` method, and the  subclass ``Mysql50ConnectionInstrumentation``  has different witnessClass